README.md

```diff
@@ -29,19 +29,26 @@ Simple example:
 int main(int argc, const char* argv[])
 {
     struct libinjection_sqli_state state;
-    int issqli;
+    injection_result_t result;
 
     const char* input = argv[1];
     size_t slen = strlen(input);
 
     /* in real-world, you would url-decode the input, etc */
 
     libinjection_sqli_init(&state, input, slen, FLAG_NONE);
-    issqli = libinjection_is_sqli(&state);
-    if (issqli) {
+    result = libinjection_is_sqli(&state);
+    
+    if (result == LIBINJECTION_RESULT_ERROR) {
+        fprintf(stderr, "error: parser encountered an error\n");
+        return -1;
```

Comment:
Returning with `-1` from `main` makes no sense. The return value must be between 0 and 255. `-1` will be cast to `255`.